docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -91,3 +81,14 @@ To revoke the access from Codacy to one or more of the OAuth providers:
 After revoking an integration, Codacy will no longer be able to access or manipulate resources that require API calls, such as detecting new pull requests or adding comments to pull requests. However, Codacy will still be able to perform operations that only require using the Git protocol either via SSH or HTTPS, such as detecting new commits and calculating diffs. To remove your repositories from Codacy and stop the analysis you must [delete them from your Codacy account](../repositories-configure/removing-your-repository.md).
 
 If you need to use an integration that you have previously revoked, log in again to Codacy with that integration so that Codacy can request the required permissions from the provider.
+
+## Why does Codacy ask for permission to create SSH keys?
+
+Codacy asks for permissions to create SSH keys because it needs to create an SSH key in your account in the following situations:
```

Comment:
Minor language tweak:

```suggestion
Codacy asks for permission to create SSH keys because it needs to create an SSH key in your account in the following situations:
```